Automattic/wp-calypso
# Remove unused styles due to the DataView updates

p1730070072537449/1730069574.185189-slack-CRWCHQGUB

## Proposed Changes

This PR addresses small refactors to remove the unused DataView overriding. 

Removed unused styles for the filter. The original PR: https://github.com/Automattic/wp-calypso/pull/89887

| no change |
|--------|
| <img width="1062" alt="Screenshot 2024-10-28 at 15 57 56" src="https://github.com/user-attachments/assets/560ae66e-ef0d-446c-9227-5997f91532b3"> | 

Removed no-longer-needed styles for cursor style. The original PR: https://github.com/Automattic/wp-calypso/pull/90449

https://github.com/user-attachments/assets/4f57ef5c-d50a-46e4-87cc-0f6a7e64d7c7

Removed styles for UI that no longer exist. The original PRs: https://github.com/Automattic/wp-calypso/pull/90842, https://github.com/Automattic/wp-calypso/pull/91663

I'll follow up with other styles soon. 

## Why are these changes being made?


*

## Testing Instructions



* Go to /sites
* Observe the table UI is not changed

## Pre-merge Checklist
